packages/flutter_tools/lib/src/localizations/gen_l10n.dart

```diff
@@ -1168,7 +1214,8 @@ class LocalizationsGenerator {
       .replaceAll('\n\n\n', '\n\n');
   }
 
-  bool _requiresIntlImport() => _allMessages.any((Message message) => message.isPlural || message.placeholdersRequireFormatting);
+  bool _requiresIntlImport() => _allMessages.any(
+          (Message message) => message.isPlural || message.isSelect || message.placeholdersRequireFormatting);
```

Comment:
Might be better to just break this down into more lines for readability:
```suggestion
  bool _requiresIntlImport() => _allMessages.any((Message message) {
    return message.isPlural 
        || message.isSelect 
        || message.placeholdersRequireFormatting;
  });
```